books/RayTracingTheNextWeek.html

```diff
@@ -1325,6 +1325,26 @@
     [Listing [perlin-interp]: <kbd>[perlin.h]</kbd> Perlin interpolation function so far]
 </div>
 
+<div class='together'>
+The output of the perlin interpretation can return negative values. Negative light isn't physically realizable (and doesn't make a lot of sense). Plus, negative values of light will be passed to the `sqrt()` function of our gamma function and get turned into `NaN`s. We will cast the perlin output back to between 0 and 1.
```

Comment:
Wrap Markdeep to 100 chars/line. Also, darkbulbs are awesome and actually useful, but yes, a problem here due to the square root.